mintlify/quickstart.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Quickstart"
+description: "Start building awesome documentation in minutes"
```

Comment:
Did you mean for this file to be included?

Reply:
nope deleting